spec/features/versions_spec.rb

```diff
@@ -246,15 +246,14 @@
     uncheck('Display social media links')
     check('Make splash page public?')
     click_button 'Save Changes'
-    splashpage_id = conference.splashpage.id
 
     click_link 'Delete'
     page.accept_alert
 
     visit admin_revision_history_path
-    expect(page).to have_text("#{organizer.name} created new splashpage with ID #{splashpage_id} in conference #{conference.short_title}")
-    expect(page).to have_text("#{organizer.name} updated public, include program, include cfp, include venue, include tickets, include lodgings, include sponsors and include social media of splashpage with ID #{splashpage_id} in conference #{conference.short_title}")
-    expect(page).to have_text("#{organizer.name} deleted splashpage with ID #{splashpage_id} in conference #{conference.short_title}")
```

Comment:
See #2703.

Reply:
But that's not it. See https://github.com/hennevogel/osem/runs/2044266325?check_suite_focus=true